### PR TITLE
(wip) assorted flavor and mechanics to support chaotic knights

### PR DESCRIPTION
**Pull request by RojjaCebolla**
as always, github is cool. feel free to commit directly to this branch/PR

### CODE REVIEW

**Comment by NullCGT** [comment]
So I have some bad news, and I'm really, really sorry to have to be the one to pass this along: Quest.txt is a relic file, and does absolutely nothing. Quest.lua is what all the quest text is pulled from.

File: dat/quest.txt [hunk 1]
```diff
@@ -1917,24 +1917,24 @@ decide if thou art ready for this great undertaking."
 "Once again, thou standest before Us, %p.  Art thou ready now?"
 %E
 %Cc Kni 00018
-"Thou disgracest this noble court with thine impure presence.  We have been
-lenient with thee, but no more.  Thy name shall be spoken no more.  We
-hereby strip thee of thy title, thy lands, and thy standing as %ca.
+"Thou disgracest this noble court with thine slovenly presence.  We have
```
Comment (NullCGT) [inline dat/quest.txt:1920]: I think we should avoid "slovenly." If I remember right, it's a pretty gendered insult towards women, and it's also a direct insult against someone's appearance. I don't think it quite fits what we're going for here, and in my opinion doesn't sound like something Arthur would say to the player.

File: dat/quest.txt [hunk 2]
```diff
@@ -1917,24 +1917,24 @@ decide if thou art ready for this great undertaking."
...
 
 "Journey forth from this place, and hone thy skills.  Return to
 Our presence when thou hast attained the noble title of %R."
 %E [You are not prepared to face %n.  Return when you are %Ra.]
 %Cc Kni 00020
-"Thou dishonourest Us, %p!  Thou hast strayed from the path of
-chivalry! Go from Our presence and do penance.  Only when thou art again
-pure mayst thou return hence."
+"Thou dishonourest Us, %p!  Thou hast strayed from thy chosen path and
+passions! Go from Our presence and contemplate.  Only when thou art again
```
Comment (NullCGT) [inline dat/quest.txt:1936]: "Thou hast strayed from thy chosen path and passions!"

I see what you're going for, with changing being a knight to being about being true to one's personal code as well as to one's order, and I think it works. I'm not sure I totally agree with the use of the word "passions" here, though. It doesn't make much sense for lawful knights, who would be expected to hold themselves above their passions, and it implies that being a chaotic knight is more about committing oneself to debauchery than following any sort of personal code.

What about this?

> Thou dishonorest Us, %p! Thou hast strayed from thy chosen path! Go from Our presence and seek reflect on thy deeds. Only when thou art again committed to thy calling mayst thou return hence."

**Comment by NullCGT** [comment]
I know I just commented on the changes here, but there's also a potential alternate solution: Give chaotic knights a completely different quest, or at the very least completely different quest dialogue.

**Comment by k21971** [comment]
> but there's also a potential alternate solution: Give chaotic knights a completely different quest, or at the very least completely different quest dialogue.

This is what I have planned for EvilHack. Not sure if I'm going to change just the quest dialogue or create entirely new quest leader/nemesis. Either way, it's going to be involved, but most likely worth it.

**Comment by NullCGT** [comment]
> > but there's also a potential alternate solution: Give chaotic knights a completely different quest, or at the very least completely different quest dialogue.
> 
> This is what I have planned for EvilHack. Not sure if I'm going to change just the quest dialogue or create entirely new quest leader/nemesis. Either way, it's going to be involved, but most likely worth it.

@k21971 I'd love to talk about that at some point! I've got some thoughts and ideas, both in terms of flavor and code.

**Comment by RojjaCebolla** [comment]
I feel like *vampire knights* are a certain thing, and it might go well with the "i'm so edgy and evil" flavor of the SporkHack patch.  They could even be Lawful, with a custom pantheon but otherwise a lot of similarities to the base role.

*Chaotic knights* of living species are another thing entirely, and should probably be themed around freedom and liberation and protecting the weak from the strong. I don't think the existing quest works super well for these.

A whole nother tack is to build on the existing NetHack system of **permaconversion**, which is a cool and underutilized system.  If permaconverting changed knights' quest and role, that might be really fun, and a *fallen* knight tells a better story than just starting out chaotic / vampiric / naughty in the first place.

@NullCGT Rather than leave this halfassed PR to sit around stale, I'm closing the PR - but the branch is still around to help us remember to address nonstandard knights in some way in the future.

**Pull request closed without merging**